test/index.js

```diff
@@ -1,36 +1,41 @@
-var bip39 = require('../')
-var download = require('../util/wordlists').download
+var bip39 = require('../');
+var download = require('../util/wordlists').download;
 var WORDLISTS = {
   english: require('../src/wordlists/english.json'),
   japanese: require('../src/wordlists/japanese.json'),
-  custom: require('./wordlist.json')
-}
+  custom: require('./wordlist.json'),
+};
 
-var vectors = require('./vectors.json')
-var test = require('tape')
+var vectors = require('./vectors.json');
+var test = require('tape');
 
-function testVector (description, wordlist, password, v, i) {
-  var ventropy = v[0]
-  var vmnemonic = v[1]
-  var vseedHex = v[2]
+function testVector(description, wordlist, password, v, i) {
+  var ventropy = v[0];
```

Comment:
More unexpected formatting changes with the latest commit. Deleted spaces, added semicolons, etc.

Reply:
fixed here: https://github.com/MetaMask/bip39/pull/1/commits/023f50a79bef68f9a72586a7e2dfbd27381c7f8f